### PR TITLE
[Feat] generate graphics in dialog boxes using modelui apis

### PR DESCRIPTION
**Pull request by nutsjian**
### What is the purpose of the change
Added modelui dialog

### Related issues/PRs

Related issues: #51 

### Brief change log
Support to generate graphics in the form of dialog boxes using modelui apis.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible 
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Comment by lepen-yang** [comment]
no problem！

File: soliduimodelui.egg-info/PKG-INFO
Comment (dlimeng) [inline soliduimodelui.egg-info/PKG-INFO:1]: soliduimodelui.egg-info module code is not submitted, this is compiled

File: workspace/bar_chart.png
Comment (dlimeng) [inline workspace/bar_chart.png:1]: Workspace is not submitted, the model generates images

**Review by lepen-yang — APPROVED** [review]
no problem

File: kernel_connection_file.json
Comment (dlimeng) [inline kernel_connection_file.json:1]: kernel_connection_file.json do not submit, compile and generate

File: app.log
Comment (dlimeng) [inline app.log:1]: app.log do not submit, compile and generate

File: solidui-web/scripts/webpack.dev.js [hunk 1]
```diff
@@ -33,6 +33,21 @@ module.exports = merge(baseConfig, {
       directory: path.join(__dirname, '../public'),
     },
 		proxy: {
+			'/solidui/models/generate': {
+				target: 'http://43.138.5.82:5110',
```
Comment (dlimeng) [inline solidui-web/scripts/webpack.dev.js]: ip fixed

File: solidui-web/scripts/webpack.dev.js [hunk 2]
```diff
@@ -33,6 +33,21 @@ module.exports = merge(baseConfig, {
...
+				changeOrigin: true,
+				pathRewrite: {}
+			},
+			'/solidui/kernel/restart': {
+				target: 'http://43.138.5.82:5010',
```
Comment (dlimeng) [inline solidui-web/scripts/webpack.dev.js]: ip fixed

File: solidui-web/scripts/webpack.dev.js [hunk 3]
```diff
@@ -33,6 +33,21 @@ module.exports = merge(baseConfig, {
...
+				changeOrigin: true,
+				pathRewrite: {}
+			},
+			'/solidui/models/api/api': {
+				target: 'http://43.138.5.82:5110',
```
Comment (dlimeng) [inline solidui-web/scripts/webpack.dev.js]: ip fixed

File: solidui-entrance/src/main/resources/application.yaml [hunk 4]
```diff
@@ -43,7 +43,7 @@ spring:
     basename: i18n/messages
   datasource:
     driver-class-name: com.mysql.jdbc.Driver
-    url: jdbc:mysql://localhost:3306/solidui?useSSL=false&useUnicode=true&characterEncoding=UTF-8&serverTimezone=Asia/Shanghai
+    url: jdbc:mysql://43.138.5.82:3306/solidui?useSSL=false&useUnicode=true&characterEncoding=UTF-8&serverTimezone=Asia/Shanghai
```
Comment (dlimeng) [inline solidui-entrance/src/main/resources/application.yaml]: replace with localhost

**Pull request closed without merging**